icu4j/.gitignore

```diff
@@ -4,4 +4,5 @@
 dependency-reduced-pom.xml
 .project
 .settings
-.classpath
\ No newline at end of file
+.classpath
```

Comment:
Looks weird, as if there is an empty line between `.classpath` and `.factorypath`

But it is not.
`.classpath` did not have a newline after it (newline at the end of file).